internal/capability/audit.go

```diff
@@ -1,17 +1,23 @@
 package capability
 
 import (
+	"context"
 	"opcap/internal/operator"
 	"strings"
 
 	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
 // Audit defines all the methods used to run a full audit Plan against a single operator
 // All new capability tests should be added to this interface and used with a capAudit
 // instance and as part of an auditPlan
 type Audit interface {
 	OperatorInstall() error
+	GetAlmExamples() error
```

Comment:
It seems that GetAlmExamples is part of the Audit initialization therefore it doesn't need to be part of the Audit interface. It seems to be much like the newCapAudit func but run after the OperatorInstall. We need figure out a more decoupled way to do this.